hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
-// Add your code here
+const input = document.querySelector('#text-input');
+const textArea = document.querySelector('#searchable-text');
+const searchableText = textArea.textContent;
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeyDown() {
+  const searchValue = input.value;
+  if (searchValue === '') {
+    textArea.textContent = searchableText;
+    return;
+  }
+  const highlightedText = searchableText.replaceAll(
+    searchValue,
+    `<mark class='font-weight-normal' style='background-color: yellow;'>${searchValue}</mark>`,
```

Comment:
Instead of using mark for highlighting the matching text I would recommend using `<span style='background-color: yellow;></span>`, as it will not make the letters spaces weirdly apart. I also don't think the font weight is needed either as it doesn't appear to be doing anything.